content/docs/guides/node-exporter.md

```diff
@@ -0,0 +1,208 @@
+go_gc_duration_seconds{quantile="0.5"} 5.846e-05
+# etc.
+```
+
+Success! The node exporter is now exposing a wide variety of system metrics that Prometheus can scrape.
+
+## Node exporter run options
+
+By default the Prometheus node exporter is available via host `:9100` and endpoint `/metrics`. You can change that using the `--web.listen-address` flag:
+
+```bash
+node_exporter --web.listen-address=9101
+```
+
+The path of the metrics-exposing endpoint (`/metrics` by default) can be changed using the `--web.telemetry-path` flag:
+
+```bash
+node_exporter --web.telemetry-path="/prometheus-metrics"
+```
+
+## Configuring your Prometheus instances
+
+Your locally running Prometheus instance needs to be properly configured in order to access node exporter metrics. The following [`scrape_config`](../prometheus/latest/configuration/configuration/#<scrape_config>) block will tell Prometheus that scrape from the node exporter via `localhost:9100`:
+
+```yaml
+scrape_configs:
+- job_name: 'node_exporter'
+  scrape_interval: 5s
+  static_configs:
+  - targets: ['localhost:9100']
+```
+
+Once Prometheus is [installed](../../introduction/first_steps) you can start it up, using the `--config.file` flag to point to the Prometheus configuration that you created:
+
+```bash
+prometheus --config.file=./prometheus.yml
+```
+
+## Exploring node exporter metrics through the Prometheus expression browser
+
+Now that Prometheus is scraping metrics from a running node exporter instance, we can explore those metrics using the Prometheus UI (aka the [expression browser](/docs/visualization/expression-browser)).
+ Navigate to `localhost:9090/graph` in your browser. Metrics specific to the node exporter are prefaced with `node_` and include metrics like `node_cpu_seconds_total` and `node_exporter_build_info`.
+
+To see all metrics available for the `node_exporter` job:
+
+```
+{job="node_exporter"}
+```
+
+The `node_filesystem_avail_bytes` metric, for example, informs you how much disk space is available on the host machine.
+
+```
+node_filesystem_avail_bytes{job="node_exporter"}
+```
+
+This will likely bring up metrics for a variety of different `device`s and `mountpoint`s. Here's an example output:
+
+```
+node_filesystem_avail_bytes{device="/dev/sda1",fstype="ext4",instance="node_exporter:9100",job="node_exporter",mountpoint="/etc/hostname"}	  15077224448
+node_filesystem_avail_bytes{device="/dev/sda1",fstype="ext4",instance="node_exporter:9100",job="node_exporter",mountpoint="/etc/hosts"}	      15077224448
+node_filesystem_avail_bytes{device="/dev/sda1",fstype="ext4",instance="node_exporter:9100",job="node_exporter",mountpoint="/etc/resolv.conf"}	15077224448
+node_filesystem_avail_bytes{device="none",fstype="aufs",instance="node_exporter:9100",job="node_exporter",mountpoint="/"}	                    15077224448
+```
+
+This is just one example, and there are many more node exporter metrics to explore.
+
+## Enabling and disabling node exporter metrics
```

Comment:
I don't think this is necessary detail for a first guide, this is intermediate to advanced stuff.

What users will care about is the key cpu/ram/disk/disk io/network/memory metrics and how to use them.

Reply:
What, in your estimation, are some of the most important metrics? We might as well provide a "maybe check these out first" list.

Reply:
The important ones are all enabled by default already.